logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -93,6 +93,7 @@ interface UserConfigRepository {
     suspend fun getSupportedProtocols(): Either<StorageFailure, Set<SupportedProtocol>>
     fun setConferenceCallingEnabled(enabled: Boolean): Either<StorageFailure, Unit>
     fun isConferenceCallingEnabled(): Either<StorageFailure, Boolean>
+    fun isConferenceCallingEnabledFlow(): Flow<Either<StorageFailure, Boolean>>
```

Comment:
I think we have a convention name of `fun`s that returns `Flow` should start with "observe", in other case - "get" / "is"

Reply:
renamed! I've seen some other `is....Flow` and followed those, but probably they are old ones - thanks for headsup!